CODING_STANDARDS.md

```diff
@@ -68,9 +72,97 @@ prefer sticking to a _single_ API for accomplishing something.
 
 ### TypeScript
 
-#### Provide function descriptions
-For functions that are more complicated than a simple getter/setter, provide at least a brief 
-sentence explaining what the function does and/or _why_ it does something.
+#### Typing
```

Comment:
We should mention that, even though TypeScript infers it, we should include the method's return type for Dgeni's sake.